framework/src/task_manager/task_manager_core.c

```diff
@@ -591,59 +605,173 @@ int taskmgr_get_handle_by_pid(int pid)
 	return TM_UNREGISTERED_APP;
 }
 
+static int taskmgr_check_broad_msg(int msg)
+{
+	if (tm_broadcast_msg[msg -1] == msg) {
+		return OK;
+	}
+	return TM_UNREGISTERED_MSG;
+}
+
+static tm_broadcast_t *taskmgr_search_broadcast_info(int msg, int handle)
+{
+	int ret;
+	sq_entry_t curr;
+
+	ret = taskmgr_check_broad_msg(msg);
+	if (ret == TM_UNREGISTERED_MSG) {
```

Comment:
call function directly here, ret uses only here.
Also MSG type is unregistered, put some error logs.